Source/UI/Coordinators/PasswordCoordinator.swift

```diff
@@ -241,6 +242,11 @@ extension PasswordCoordinator {
             // Fallback to passsword login
             return nil
         }
+
+        let hasLoggedInBefore = Settings.value(forKey: Constants.HasLoggedInBeforeSettingsKey) as? Bool
```

Comment:
```
guard let hasLoggedInBefore = Settings.value(forKey: Constants.HasLoggedInBeforeSettingsKey) as? Bool else { return nil }
```

Reply:
You can also join with the previous guard
` guard #available(iOS 11.3, *) else {`

Reply:
Wouldn't this prevent users from reusing password after app upgrade?